src/moapplication.cpp

```diff
@@ -390,8 +390,14 @@ void MOApplication::externalMessage(const QString& message)
       }
     }
   } else if (isNxmLink(message)) {
-    MessageDialog::showMessage(tr("Download started"), qApp->activeWindow(), false);
-    m_core->downloadRequestedNXM(message);
+    if (InstanceManager::singleton().currentInstance()->gamePlugin() == nullptr) {
+      // This can happen if MO2 is started with the --pick option and no instance has
+      // been selected yet, in which case m_core will be null.
+      reportError(tr("No instance has been selected. Cannot download mod."));
```

Comment:
Can you change the message to something along `"You need to select an instance before trying to download mods."`, I think it'd be better.

Reply:
```suggestion
      reportError(tr("You need to select an instance before trying to download mods."));
```